<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomepageWindow</class>
 <widget class="QMainWindow" name="HomepageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>525</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>HomepageWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="separator">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="separator_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="separator_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>50</y>
      <width>961</width>
      <height>631</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="homepage">
     <widget class="QLabel" name="systemThroughput">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Throughput Green (tickets/hour)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="greenThroughputLineEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>380</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>420</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Throughput Red (tickets/Hour)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="redThroughputLineEdit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>420</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="occupancyTableWidget">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>371</width>
        <height>201</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Train</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Location</string>
       </property>
      </column>
     </widget>
     <widget class="QTextEdit" name="trackFailureAlertTextEdit">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>20</y>
        <width>261</width>
        <height>71</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="totalOccupancyLineEdit">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>230</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>230</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Occupancies: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>260</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of Dispatched Trains</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dispatchedTrainsCountLineEdit">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>260</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="timeHourLineEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="timeMinuteLineEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>19</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Time: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Current Line:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="trackLineLineEdit">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mangeTrainsPage">
     <widget class="QPushButton" name="uploadScheduleButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>340</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Upload Schedule</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="trainTableWidget">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>30</y>
        <width>501</width>
        <height>241</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Train</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Destination</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Arrival Time</string>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Departure Time</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>S. Speed(mph)</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="plusButton">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>280</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="minusButton">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>280</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="scheduleTrainLineEdit">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>280</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter Train #</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="scheduleDestinationLineEdit">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>280</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter Destination</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="scheduleArrivalLineEdit">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>280</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Enter Arrival Time</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="autoDispatchCheckBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>0</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Auto-dispatch</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>290</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="dispatchButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>340</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Dispatch</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Time: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>19</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="timeMinuteLineEdit_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="timeHourLineEdit_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="manageTracksPage">
     <widget class="QTableWidget" name="trackSectionTable">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>60</y>
        <width>101</width>
        <height>192</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Closed Tracks</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="switchTableWidget">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>60</y>
        <width>91</width>
        <height>192</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Switches</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="addTrackSectionButton">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>290</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeTrackButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>290</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>-160</y>
        <width>1021</width>
        <height>761</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>green_and_red_track_small2.png</pixmap>
      </property>
     </widget>
     <widget class="QLineEdit" name="trackLineEdit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>260</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="maintenanceModeCheckBox">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>10</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Maintenance Mode</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addSwitchButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>290</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeSwitchButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>290</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="switchLineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>260</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Time: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>50</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>19</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="timeMinuteLineEdit_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="timeHourLineEdit_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="testInterfacePage">
     <widget class="QComboBox" name="testTrackComboBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>80</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Select Track</string>
      </property>
     </widget>
     <widget class="QLabel" name="trackInputLabel">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Track Input</string>
      </property>
     </widget>
     <widget class="QComboBox" name="testTrackConditionComboBox">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Select Track</string>
      </property>
     </widget>
     <widget class="QLabel" name="trackInputLabel_2">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>30</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter Train Input</string>
      </property>
     </widget>
     <widget class="QComboBox" name="testLocationComboBox">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>100</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Select Location</string>
      </property>
     </widget>
     <widget class="QComboBox" name="testTrainComboBox">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>60</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Select Train</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="testSpeedLineEdit">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>140</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter Current Speed</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="testAuthorityLineEdit">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>180</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter Authority</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>60</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Train:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>110</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Location:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>150</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Speed:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>180</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Authority:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submitTestTrackInputButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>180</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submitTestTrainInputButton">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>250</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Track:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>120</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Condition:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="testDestinationLineEdit">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>210</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Enter Destination</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>210</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Destination:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submitBlockButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>320</y>
        <width>80</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>submit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="startBlockLineEdit">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="destinationBlockLineEdit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>270</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="homepageButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Homepage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manageTrainsButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Manage Trains</string>
    </property>
   </widget>
   <widget class="QPushButton" name="manageTracksButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Manage Tracks</string>
    </property>
   </widget>
   <widget class="Line" name="separator_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="testInterfaceButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>0</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Interface</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
